mta-v7.x/mta.sw.yaml

```diff
@@ -55,6 +57,14 @@ functions:
   - name: logInfo
     type: custom
     operation: "sysout:INFO"
+  - name: countMandatoriesIncidents
+    type: expression
+    operation: .applicationAnalysis | [select (.issues[].category=="mandatory")][0] | [.issues[]?.incidents //[]] | .[-1][-1].id
+    # cat anal.json | jq '. | [select (.issues[].category=="mandatory")][0] | [.issues[]?.incidents //[]] | .[-1][-1].id'
```

Comment:
should the comment remain or be removed?

Reply:
oh yes, thx